### DISCUSSION TITLE
google drive 挂载  客户端ID     客户端密钥

### DISCUSSION THREAD

**Discussion by shaojs321** [AlistGo/alist#841] [Q&A]
<img width="877" alt="Snipaste_2022-03-28_21-43-21" src="https://user-images.githubusercontent.com/56724141/160411139-44b61f4b-b964-4dcd-bb00-0a6f2381bbf3.png">

使用自己的 客户端id 客户端密钥，刷新令牌经常失效，需要重新填写，
alist的google drive  是不是必须填客户端id 客户端密钥  用公共api的话，这两个该填什么呢

**Comment by xhofe** [comment]
用什么填什么。
